src/authors/authors.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1>Lists of Authors</h1>
+    <div class="wrap">
+         
+     <div class="box">
+            <p>Justyna Rozbicka ...</p>
+            <p class="gitHub">  <a href="#">GitHubLink</a></p>
```

Comment:
Why wrap `<a>` in `<p>`? Just add to the `<a>` class. Besides I suggest naming the class `link--github`. Read more about using `BEM`:

http://getbem.com/

Reply:
`<a href="REAL_LINK" target ="_blank">...</a>`